subprojects/docs/src/docs/userguide/signingPlugin.xml

```diff
@@ -101,6 +101,84 @@ gradle.taskGraph.whenReady { taskGraph ->
         </section>
     </section>
 
+    <section id="sec:using_gpg_agent">
+        <title>Using gpg-agent</title>
+        <para>
+            By default the signing plugin uses a Java-based implementation of PGP for signing. This implementation cannot
+            use the gpg-agent program for managing private keys, though. If you want to use the gpg-agent, you can change
+            the signatory implementation used by the signing plugin to
+            <apilink class="org.gradle.plugins.signing.signatory.gnupg.GnupgSignatory" />.
+        </para>
+        <sample id="use-gnupg" dir="signing/gnupg-signatory" title="Sign with GnuPG">
+            <sourcefile file="build.gradle" snippet="configure-signatory" />
+        </sample>
+        <para>
+            This tells the signing plugin to use the <literal>GnupgSignatory</literal>
+            instead of the default <apilink class="org.gradle.plugins.signing.signatory.pgp.PgpSignatory" />.
+            The <literal>GnupgSignatory</literal> relies on the gpg2 progam to sign the artifacts. Of course, this requires
+            that GnuPG is installed.
```

Comment:
It doesn't look like that example does what we want, it rather seems like the author has confused PGP and GnuPG. SBT also [mentions shelling out to `gpg`][sbt] as an alternative to using BouncyCastle. The closest thing I've managed to find is [GnuPG for Java][gpg4j] (linked from [GnuPG][gpg-libraries]), which is 7 years old, undocumented and not published on Maven Central.

[sbt]: https://github.com/sbt/sbt-pgp/wiki/Using-Bouncy-Castle
[gpg4j]: https://github.com/smartrevolution/gnupg-for-java
[gpg-libraries]: https://gnupg.org/software/libraries.html